pages/about/history.vue

```diff
@@ -235,7 +271,20 @@ export default {
 </script>
 
 <style scoped>
-.paragraph-title {
-    @apply font-serif text-pink-500;
+.history__paragraph-title {
+    @apply mt-1 mb-1 font-serif font-bold text-left text-pink-700;
+    font-size: 28px;
+}
+.history__paragraph-content {
+    @apply mt-1 mb-9 leading-5 md:leading-8 last:mb-0;
+    font-size: 16px;
+}
+@media (min-width: 768px) {
+    .history__paragraph-title {
+        font-size: 32px;
+    }
```

Comment:
spec updated during design reviewing

```suggestion
.history__paragraph-title {
    @apply mt-1 mb-1 font-serif font-bold text-left text-pink-700;
    font-size: 24px;
}
.history__paragraph-content {
    @apply mt-1 mb-9 leading-5 md:leading-8 last:mb-0;
    font-size: 16px;
}
@media (min-width: 768px) {
    .history__paragraph-title {
        font-size: 28px;
    }
```